Sources/SwiftProtobuf/Google_Protobuf_Duration+Extensions.swift

```diff
@@ -130,18 +132,41 @@ extension Google_Protobuf_Duration: _CustomJSONCodable {
     }
 }
 
-
 extension Google_Protobuf_Duration: ExpressibleByFloatLiteral {
     public typealias FloatLiteralType = Double
 
     public init(floatLiteral value: Double) {
+        // TODO: this rounds the nanos part towards zero, not to the nearest
```

Comment:
So if the developer types a TimeInterval of 1.9999999999 will that stay or get rounded to 2.0 also?